atomate/vasp/firetasks/parse_outputs.py

```diff
@@ -753,6 +755,300 @@ def run_task(self, fw_spec):
         logger.info("Thermal expansion coefficient calculation complete.")
 
 
+@explicit_serialize
+class MagneticOrderingsToDB(FiretaskBase):
```

Comment:
We should talk about the "builder" vs "firework" way to do this

Honestly I think builders should be used for essentially all these problems. The strategy of running a single Firework once at the end of a workflow, and assuming things are not going to change later, is just going to cause problems down the line.

Maybe schedule a phone call to discuss?

Reply:
Yes, I was following the previous examples in atomate for this.

My judgement was that the atomate workflow should work as a stand-alone workflow, which is what this task is for: if someone wants to just understand magnetic orderings they can run the workflow and get a nice collection as output.

However, for MP, we will need a builder regardless, and there does seem to be some duplication of effort here. Happy to discuss further!